plugin-server/src/cdp/services/hog-function-manager.service.ts

```diff
@@ -130,6 +131,7 @@ export class HogFunctionManagerService {
             SELECT ${HOG_FUNCTION_FIELDS.join(', ')}
             FROM posthog_hogfunction
             WHERE deleted = FALSE AND enabled = TRUE AND type = ANY($1)
+            ORDER BY execution_order NULLS LAST
```

Comment:
Oh cool - I hadn't thought about doing it that way. That said however - doesn't work 😅 

This will only work when we reload all hog functions which is only a background timer. 

We also have individual reloads whenever a function changes. These ones only load the one function that changed so we would likely need to do the sorting in code and make sure it applies when any reload occurs. Tests too to prove it works...